cmd/bucket-handlers_test.go

```diff
@@ -580,7 +580,7 @@ func testListBucketsHandler(obj ObjectLayer, instanceType, bucketName string, ap
 			expectedRespStatus: http.StatusOK,
 		},
 		// Test case - 2.
-		// Test case with invalid accessKey to produce and validate Signature MisMatch error.
+		// Test case with invalid accessKey to produce and validate Signature MissMatch error.
```

Comment:
typos doesn't catch case sensitive mistakes

Reply:
like?

Reply:
@poornas ?

Reply:
s/MisMatch/Mismatch/ should fix this - no need to camelcase this.

Reply:
MissMatch is not right here @harshavardhana